packages/generators/admin/component/templates/tests/reducer.test.js.hbs

```diff
@@ -0,0 +1,21 @@
+// import produce from 'immer';
+import {{camelCase name}}Reducer, { initialState } from '../reducer';
+import { fixtures } from '../../../../../../../../admin-test-utils';
+// import { someAction } from '../actions';
+
+/* eslint-disable default-case, no-param-reassign */
+describe('{{camelCase name}}Reducer', () => {
+  let state;
+  beforeEach(() => {
+    state = {
+      ...fixtures.store.state,
+      '{{plugin}}_{{camelCase name}}': initialState,
+    };
+  });
+
+  it('returns the initial state', () => {
+    const expectedResult = state;
```

Comment:
Thanks, it's changed 👍

Btw, some plugin names contains `-` (like `content-type-builder`) which mean we need to do `state['{{plugin}}_{{ camelCase name }}']`. But eslint will complain since in most cases the dot notation will be preferred.

Just saying that here because it's related to another comment you made about the `import/order` rule, trying to generate already linted files will probably be painful xD
Thus I added a custom action type to automatically fix linting errors and keep templates simple, I hope it's ok.